lib/bosh/workspace/merge_tool.rb

```diff
@@ -0,0 +1,76 @@
+module Bosh::Workspace
+  class MergeTool
+    include Bosh::Exec
+
+    attr_accessor :name, :version
+
+    def initialize(merge_tool = nil)
+      @name, @version = case merge_tool
+      when Hash
```

Comment:
Could you move the schema validations into [schemas/project_deployment.rb](https://github.com/cloudfoundry-incubator/bosh-workspace/blob/master/lib/bosh/workspace/schemas/project_deployment.rb)?